packages/genealogytree_test.tex

```diff
@@ -0,0 +1,21 @@
+\documentclass{report}
+\usepackage{genealogytree}
+\begin{document}
+    \begin{tikzpicture}
+        \genealogytree[template=signpost]{
```

Comment:
Hi @patbro, I sure could try another template, but then, since the package is out of scope, this would be just to tick the build box. The local packages are working great, but unfortunatelly I'm currently blocked with using bazel-latex, due to the pdf stuff that you've mentioned recently on a different issue.

Cheers!